src/qtiCommonRenderer/renderers/interactions/MediaInteraction.js

```diff
@@ -156,21 +149,21 @@ var render = function render(interaction) {
         }
 
         //initialize the component
-        $container.on('responseSet', function() {
+        $container.on('responseSet', function () {
```

Comment:
Why did you keep anonymous function here?

Reply:
good catch! I forgot it